build.sh

```diff
@@ -14,6 +14,7 @@ npm ci
 rm -Rf dist build
 rm -Rf node_modules/enketo-core/node_modules/
 patch -f node_modules/enketo-core/src/js/Form.js < node_modules/cht-core-3-11/webapp/patches/enketo-inputs-always-relevant.patch
+patch -f node_modules/enketo-core/src/js/page.js < patches/enketo-handle-no-active-pages.patch
```

Comment:
does CHT do this?

Reply:
Core has been doing this since [Sep 2019](https://github.com/medic/cht-core/pull/5941/files#diff-8e7e5f941c67b0cf481a9d650295119d4922b96117e7adc069ba30a81a78c65fR597)

Same patch is in [master](https://github.com/medic/cht-core/blob/master/Gruntfile.js#L565)

The only difference is that I remove the console.warning since harness tracks those are errors